scripts/generateReactIcons.ts

```diff
@@ -1,19 +1,19 @@
 /* eslint-disable @typescript-eslint/ban-ts-comment */
-import fastGlob from "fast-glob";
-import { join } from "path";
-import { writeFile, readFile } from "fs/promises";
-import { camelCase, toUpper } from "lodash";
+import fastGlob from 'fast-glob';
```

Comment:
![Screenshot 2022-08-05 at 08 49 37](https://user-images.githubusercontent.com/3276065/183018655-8906252b-0733-4a58-92e1-8118aeddb5c2.png)

This error has a false negative because the script is not added to the typescript scope. We can solve the issues by adding `"scripts/**/*"` to include property in tsconfig.json. This will unveil further code smells, let's tackle them in a pair-programming session 👌

Reply:
solved in pp session